tools/clang/test/HLSLFileCheck/hlsl/workgraph/case070_noinput.hlsl

```diff
@@ -53,6 +59,22 @@ void node070_thread() { }
 // CHECK-NOT: i32 20, i32 {{![-=9]+}}
 // CHECK-SAME: }
 
+// Metadata for node070_mesh
```

Comment:
I think we should put mesh node tests in a separate file from other node types if possible.

Reply:
Done